src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/SecondFactorProjector.php

```diff
@@ -154,10 +154,11 @@ public function applyGssfPossessionProvenAndVerifiedEvent(GssfPossessionProvenAn
 
     public function applyEmailVerifiedEvent(EmailVerifiedEvent $event): void
     {
-        $unverified = $this->unverifiedRepository->find($event->secondFactorId->getSecondFactorId());
-        if (is_null($unverified)) {
+        if ($event->secondFactorType->isU2f()) {
+            // u2f is deprecated so those events shouldn't be handled anymore
             return;
         }
+        $unverified = $this->unverifiedRepository->find($event->secondFactorId->getSecondFactorId());
```

Comment:
Harsh change in behavior, I do not see that the find method now throws an exception, or otherwise exits applying the event when the unverified token is not found in the projection.

Reply:
When a replay is done with an event stream with U2F events this resulted in an exception. Because U2F tokens weren't added anymore.